content/post/why-i-joined-sourced-francesc.md

```diff
@@ -0,0 +1,117 @@
+many other repositories are not on GitHub: what about the Unix kernel?
+
+### So what does source{d} do?
+
+So what does source{d} do? They ... We! provide a powerful platform to access all of this
+data in an easier and more powerful way.
+
+Rather than limiting repositories on GitHub, source{d} is able to analyze any code
+repository in the world, including those that are not even on the internet by running
+our open source software on your own premise.
+We're very proud of the pipeline we're building to ingest every public git repo in the world.
+
+Secondly, we consider that the input to many good analysis should be the abstract syntax
+tree of a program rather than the flat suite of bytes that is the source code.
+We believe this so much that we've created [Babelfish](https://doc.bblf.sh/), a project that
+one day will be able to parse any programming language and generate an abstract syntax
+tree in a universal format. We call this format a universal abstract syntax tree or UAST.
+
+Finally, while we love [regular expressions](https://xkcd.com/208/), we believe that Machine Learning will
+revolutionize how we analyze programs. There's a never-ending list of use cases that could benefit from ML
+over source code: autocompletion (that doesn't require a connection to a 3rd party server),
+code linters, architecture analyzers, automated code reviews, and (one day) source code generation
+from unit tests or even natural language specifications.
+
+If you're curious, you should go see our [papers](https://arxiv.org/abs/1704.00135) and
+[blog posts](https://blog.sourced.tech/) on these topics.
+
+What's my favorite part of the company? The incredibly incredibly talented team tackling
+problems at different layers:
+
+- retrieval: let's fetch all of the code in the world.
+- engine: creating a declarative API to access all of that code.
+- ML: building and training models that can power complex applications.
+- applications: finding the use cases that allows us to prove the power of the platform, while providing as much benefit to the community as possible.
+
+![we want you](img/hello-sourced/wewantyou.jpg)
+
+But the applications team at source{d} is not the only one building applications,
+because we're building a platform. We want to empower every developer with access
+to the largest code dataset and the most advanced ML tools, so with our tech and your
+creativity we can improve how fifty million developers in the world write code.
+
+### So what's coming?
+
+As VP of Developer Relations my job is to strategize how source{d} can empower developers
+all around the world to write better code by:
+
+- writing new models that will power amazing tools,
+- building tools powered by ML on code, and of course
+- using those developer tools to analyze and improve source code.
+
+source{d} is building a platform by *developers* where *developers* can build tools for *developers* [[1]](https://www.youtube.com/watch?v=KMU0tzLwhbE).
```

Comment:
what if we embed the video? would it be too funny? :P

Reply:
I think it's better to keep it as a surprise, people click without knowing what they're gonna get and...

![steve ballmer](https://media1.tenor.com/images/3303d5de4f7cd7b79d88d170fbe0a7fc/tenor.gif?itemid=4349581)